From 8d04847f5a0c841dd000877b5abf3ff5aa902293 Mon Sep 17 00:00:00 2001
From: James Slagle <jslagle@redhat.com>
Date: Thu, 13 Feb 2014 17:09:46 -0500
Subject: [PATCH 09/12] Common cinder install code

Move common cinder install code that is needed for both the cinder
package and source installs into a common script that gets used.

Change-Id: If46a9d3b6777b327ae11abf8b4f3d223cc147a2d
---
 elements/cinder/install.d/73-cinder                       | 9 +++++++++
 elements/cinder/install.d/cinder-source-install/72-cinder | 6 ------
 2 files changed, 9 insertions(+), 6 deletions(-)
 create mode 100755 elements/cinder/install.d/73-cinder

diff --git a/elements/cinder/install.d/73-cinder b/elements/cinder/install.d/73-cinder
new file mode 100755
index 0000000..d59826c
--- /dev/null
+++ b/elements/cinder/install.d/73-cinder
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -eux
+
+mkdir -p /etc/tgt/conf.d
+install -d -o root -g cinder -m 0770 /var/lib/cinder/volumes
+register-state-path /var/lib/cinder
+echo 'include /etc/tgt/conf.d/cinder_tgt.conf' > /etc/tgt/targets.conf
+echo 'include /mnt/state/var/lib/cinder/volumes/*' > /etc/tgt/conf.d/cinder_tgt.conf
diff --git a/elements/cinder/install.d/cinder-source-install/72-cinder b/elements/cinder/install.d/cinder-source-install/72-cinder
index af6d964..9530364 100755
--- a/elements/cinder/install.d/cinder-source-install/72-cinder
+++ b/elements/cinder/install.d/cinder-source-install/72-cinder
@@ -5,12 +5,6 @@ install-packages lvm2 libssl-dev tgt
 echo "qpid-python" | tee -a /opt/stack/cinder/requirements.txt
 os-svc-install -n cinder -u cinder -r /opt/stack/cinder -c cinder-all
 
-mkdir -p /etc/tgt/conf.d
-install -d -o root -g cinder -m 0770 /var/lib/cinder/volumes
-register-state-path /var/lib/cinder
-echo 'include /etc/tgt/conf.d/cinder_tgt.conf' > /etc/tgt/targets.conf
-echo 'include /mnt/state/var/lib/cinder/volumes/*' > /etc/tgt/conf.d/cinder_tgt.conf
-
 cp -a /opt/stack/cinder/etc/cinder/rootwrap* /etc/cinder
 cp /opt/stack/cinder/etc/cinder/api-paste.ini /etc/cinder/api-paste.ini
 cp /opt/stack/cinder/etc/cinder/policy.json /etc/cinder/policy.json
-- 
1.8.5.3

